src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  $root:&;
+
+  padding: $paddings;
+  display: flex;
+  flex-wrap: wrap;
+  width: calendarWidth($sideSize, $space, $paddings);
+  gap: 1px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include daySize($sideSize);
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid black;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      transition: background-color 0.5s, transform 0.5s;
```

Comment:
Missed this, but I think you should check it for #9 in checklist.

Reply:
I agree. Thanks :)